modmail/database/models/servers.py

```diff
@@ -0,0 +1,20 @@
+from typing import TYPE_CHECKING
+
+from tortoise import fields
+from tortoise.models import Model
+
+
+if TYPE_CHECKING:
+    from .configuration import Configurations
+    from .tickets import Tickets
+
+
+class Servers(Model):
```

Comment:
`class Guild(Model):`

Also the docstring below (can't comment on multiple lines for some reason)